custom_components/zwave_mqtt/services.py

```diff
@@ -0,0 +1,125 @@
+"""Methods and classes related to executing Z-Wave commands and publishing these to hass."""
+import logging
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+
+from . import const
+
+_LOGGER = logging.getLogger(__name__)
+
+BASIC_INSTANCE_SCHEMA = vol.Schema(
+    {
+        vol.Optional(const.ATTR_INSTANCE_ID, default=1): vol.Coerce(int),
+    }
+)
+ADD_NODE_SCHEMA = BASIC_INSTANCE_SCHEMA.extend(
+    {
+        vol.Optional(const.ATTR_SECURE, default=False): vol.Coerce(bool),
+    }
+)
+NODE_SCHEMA = BASIC_INSTANCE_SCHEMA.extend(
+    {
+        vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
+    }
+)
+SET_CONFIG_PARAMETER_SCHEMA = BASIC_INSTANCE_SCHEMA.extend(
+    {
+        vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
+        vol.Required(const.ATTR_CONFIG_PARAMETER): vol.Coerce(int),
+        vol.Required(const.ATTR_CONFIG_VALUE): vol.Any(vol.Coerce(int), cv.string),
+        vol.Optional(const.ATTR_CONFIG_SIZE, default=2): vol.Coerce(int),
+    }
+)
+
+
+class ZWaveServices:
+    """Class that holds our services ( Zwave Commands) that should be published to hass."""
+
+    def __init__(self, hass, manager):
+        """Initialize with both hass and ozwmanager objects."""
+        self._hass = hass
+        self._manager = manager
+
+    def register(self):
```

Comment:
Let's make this a @callback and use `self._hass.services.async_register` That way you don't need to call it with `hass.async_add_job`.